src/netstandard/ref/System.Buffers.cs

```diff
@@ -120,6 +134,51 @@ public abstract partial class ReadOnlySequenceSegment<T>
         public long RunningIndex { get { throw null; } protected set { } }
     }
     public delegate void ReadOnlySpanAction<T, in TArg>(System.ReadOnlySpan<T> span, TArg arg);
+    public ref partial struct SequenceReader<T> where T : struct, System.IEquatable<T>
```

Comment:
The generic constraint for `SequenceReader` should be `unmanaged`:

`public ref partial struct SequenceReader<T> where T : unmanaged, IEquatable<T>`

https://github.com/dotnet/corefx/blob/master/src/System.Memory/ref/System.Memory.cs#L405

However GenAPI doesn't yet support the unmanaged constraint.

Reply:
That's a concern. I'll check for other `unmanaged` annotations.

Reply:
Check that no other APIs are affected.